Lib/test/test_functools.py

```diff
@@ -210,6 +210,50 @@ def foo(bar):
         p2.new_attr = 'spam'
         self.assertEqual(p2.new_attr, 'spam')
 
+    def test_placeholders_trailing_trim(self):
```

Comment:
Add test with functions that have default values and with/wo placeholders.

Reply:
This is not relevant anymore. I decided to prohibit trailing `Placeholders`